packages/provider-utils/src/generate-id-custom-alphabet.ts

```diff
@@ -3,7 +3,7 @@
 //
 // MIT License
 //
-// Copyright (c) 2020–2024 AI
+// Copyright (c) 2017 Andrey Sitnik <andrey@sitnik.ru>
```

Comment:
honestly I have no idea where I got the `2020–2024 AI` from, I thought I copied it from nanoid's license text